### PR TITLE
Use the UserPosts table to populate posts

### PR DESCRIPTION
**Pull request by ghostwriternr**
The schema already had a UserPosts table ( my bad - the schema diagram was using a different name from the code 😞 ). Have updated the code to re-use that table instead of create a new one. Can you please review this PR? If the structure and usage look good to you, please merge this into your branch and work on top of this.

NOTE: I have *not* tested this in my local, so expect errors 🙈 Have some troubles with setting up the dbs and in a time crunch currently.

### CODE REVIEW

File: c3po/job/utils/metadata.py [hunk 1]
```diff
@@ -1,38 +1,58 @@
...
+
+def insert_metadata(data):
+    url = data["link"]
     with session_scope() as session:
         data = SongData(url)
-        new_link = _insert_post(url, user, date_time, session)
-        if(new_link):
+        # This is a placeholder for until we can fetch real user details
+        user = _insert_default_user(session)
+        new_link = _insert_post(url, user, data, session)
```
Comment (thescriptninja) [inline c3po/job/utils/metadata.py:18]: The data being passed to the `_insert_post()` method is a SongData object, whereas a JSON response from MongoDB is to be passed. Maybe we can have two variables: `raw_data` and `data`?

**Comment by thescriptninja** [comment]
Also, can we change the type of the `share_date` column in `UserPosts` to Datetime? It should provide better flexibility while querying I guess.

**Comment by thescriptninja** [comment]
The changes are pretty minor I think. Can I push to this branch?

File: c3po/job/utils/metadata.py (hunk 1 above)
Comment (ghostwriternr) [reply c3po/job/utils/metadata.py:18]: Oh ow. Yes, that's definitely needed! Didn't notice the variable re-use.

**Comment by ghostwriternr** [comment]
Yes please!

**Pull request merged**